components/decimal/src/provider.rs

```diff
@@ -85,12 +85,12 @@ pub struct GroupingSizesV1 {
 #[derive(Debug, PartialEq, Clone, yoke::Yokeable, zerofrom::ZeroFrom)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[cfg_attr(feature = "datagen", derive(serde::Serialize))]
-#[zerovec::make_varule(DecimalSymbolsV1StringsULE)]
+#[zerovec::make_varule(DecimalSymbolsV1Strs)]
 #[zerovec::derive(Debug)]
 #[zerovec::skip_derive(Ord)]
 #[cfg_attr(feature = "serde", zerovec::derive(Deserialize))]
 #[cfg_attr(feature = "datagen", zerovec::derive(Serialize))]
-pub struct DecimalSymbolsV1Strings<'data> {
+pub struct DecimalSymbolsV1StrsBuilder<'data> {
```

Comment:
Thought: `DecimalSymbolStrsBuilder`? Do we need the `V1` since it is not itself a data struct?

Reply:
oh good point. I didn't want to mess with the loading for the benchmark but it's totally fine here!

Reply:
I think this reply is to a different comment